apps/desktop/jest.config.mjs

```diff
@@ -0,0 +1,7 @@
+import config from "@umami/jest-config";
+
```

Comment:
Wondering why not ts?

Reply:
It used to be a typescript file. And all the perks you get with typescript compiler included